src/data/feature_position_map.js

```diff
@@ -79,6 +82,14 @@ export default class FeaturePositionMap {
     }
 }
 
+export function getNumericId(value: mixed) {
+    const numValue = +value;
```

Comment:
I think this incorrectly handles strings that can be converted to javascript numbers but not javascript integers:
- integers javascript can't safely represent `'9007199254740993'`
- decimals `'1.2'`
- `'Infinity'`

It also looks like users of this function expect integers but this can return a float if given a float. Is that ok because it should never be given a float?

Reply:
Initially this was hashing all non-integer ids, but then I changed it because I thought there's no longer a need to have it limited to integer — those get saved in a Float64Array anyway, which supports decimals, values like `Infinity`, and values bigger than `MAX_SAFE_INTEGER`.

Reply:
> It also looks like users of this function expect integers 

Oh, actually, in addition to the above, this function being exported is a leftover from earlier — how things get stored in the position map should be an internal detail so there won't be "users" of this. I'll remove the "export".

Reply:
> those get saved in a Float64Array anyway, which supports decimals, values like Infinity, and values bigger than MAX_SAFE_INTEGER

Neither javascript numbers (which are 64 bit floats) or Float64Array can store integers > MAX_SAFE_INTEGER safely. Both `'9007199254740993'` and `'9007199254740992'` will look like the same id to this implementation. This might seem unlikely but we ran into this with iD editor because osm uses sequential 64 integers as ids.

Similar problems theoretically exist with decimals but seem less likely.

Reply:
Fair enough — updated to hash as a string in case a numeric id is more than `MAX_SAFE_INTEGER`.